marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -173,7 +173,7 @@ class Flow {
     let stepDetails = this.step.getStepByNameAndType(stepRef.stepDefinitionName, stepRef.stepDefinitionType);
 
     //here we consolidate options and override in order of priority: runtime flow options, step defined options, process defined options
-    let combinedOptions = Object.assign({}, stepDetails.options, stepRef.options, options);
+    let combinedOptions = Object.assign({}, stepDetails.options, flow.options, stepRef.options, options);
```

Comment:
There is another place where we combine options line 262 in runStep() method in impl/flow.sjs

Reply:
That should be removed. At that point, the options have already been combined.